extensions/amazon-lambda/common-runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AbstractLambdaPollLoop.java

```diff
@@ -35,6 +35,10 @@ public AbstractLambdaPollLoop(ObjectMapper objectMapper, ObjectReader cognitoIdR
         this.launchMode = launchMode;
     }
 
+    protected boolean shouldLog(Exception e) {
```

Comment:
Always true, shouldn't this check listed exceptions?

Reply:
never mind!

Reply:
😛 Yeah, I wasn't sure if the behavior made sense in the Funqy and REST implementations...